.eslintrc.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
   ],
   rules: {
     // VS Code linting will not respect the `.prettierrc` options unless injected here:
-    'prettier/prettier': ['error', prettierrc],
+    'prettier/prettier': ['warn', prettierrc],
```

Comment:
good open source PRs try to limit extraneous changes. I appreciate the Node update but can you explain why this change was made? there could be perfectly valid reasons, but without mention it's hard as a reviewer to know

Reply:
When Marty and I were editing the file we just didn't want the red squigglies to make it seem like it was broken when it was just prettier. I think I can change that back